esprima.js

```diff
@@ -4230,6 +4233,18 @@
                 }
 
                 lex();
+
+                if (match(')')) {
+                    possibleArrow = lookahead2();
+                    /* istanbul ignore else */
```

Comment:
Perhaps `foo(a, (b),)`? I'm not familiar enough either but the arrow function would look like `foo(a, (b) => {b+1},)`.